host/4/republish.yml

```diff
@@ -135,6 +135,18 @@ steps:
     displayName: tag and push dotnet8-isolated images
     continueOnError: false
 
+ - bash: |
```

Comment:
Just make sure that this pipeline could still pick up the dotnet7 mariner image. IE - The output image name has not changed

Reply:
Yep, this change only affecsts the path to the file and the display name for the tests: ![image](https://github.com/Azure/azure-functions-docker/assets/75647613/1a26d523-db5c-4e3e-8f5e-1445f323ab52)